templates/cli/lib/commands/pull.js.twig

```diff
@@ -27,6 +30,64 @@ const pullProject = async () => {
     success();
 }
 
+const pullFunctions = async ({ all, yes } = {}) => {
+    let functions = [];
+    let questions = questionsPullFunctions;
+
+    const localFunctions = localConfig.getFunctions();
+
+    if (all) {
+        questions = yes ? [] : questionsPullFunctions[1];
+        functions = (await paginate(functionsList, { parseOutput: false }, 100, 'functions')).functions;
+    }
+
+    const answers = await inquirer.prompt(questions);
+
+    const overridingLocalChanges = yes ?? answers.override.toLowerCase() === "yes";
+    const selectedFunctions = functions.length === 0 ? answers.functions : functions;
+
+    for (let func of selectedFunctions) {
+        const functionExistLocally = localFunctions.find((localFunc) => localFunc['$id'] === func['$id']) !== undefined;
+
+        if (!overridingLocalChanges && functionExistLocally) {
+            log(`Skipping locally found implementation of ${func['name']}`)
+            continue;
+        }
+        if (functionExistLocally) {
+            localConfig.updateFunction(func['$id'], func);
+        } else {
+            func['path'] = `functions/${func['$id']}`;
+            localConfig.addFunction(func);
+        }
+
+        const localFunction = localFunctions.find((localFunc) => localFunc['$id'] === func['$id']);
+
+        if (localFunction['deployment'] === '') {
+            continue
+        }
+
+        const compressedFileName = `${+new Date()}.tar.gz`
```

Comment:
Should this be a parameter? Maybe it doesn't matter

Reply:
This is just a temp name, and the file get deleted after it is extracted.